### DISCUSSION TITLE
Donations on Behalf of Someone

### DISCUSSION THREAD

**Discussion by SudharakaP** [opencollective/opencollective#4851] [Ideas]
@opencollective/core : This is something that occurred to me a few days back. I am wondering if it's useful to add support to donate on behalf of someone. So currently in our contribution flow one could donate to a collective or some cause they support but the donation is always tied to the donor. What I am suggesting is say I could donate on behalf of some other person or occasion. For example take a look at; 

https://support.cancer.ca/site/SPageServer?pagename=DJ_NW_donation_type_selector&s_locale=en_CA#_ga=2.139400947.1190799814.1634828065-1601835427.1634828064

or

https://donate.bccancerfoundation.com/site/Donation2;jsessionid=00000000.app20047a?df_id=5183&5183.donation=form1&mfc_pref=T&NONCE_TOKEN=53B17937875AAF68BD292E242B8C9558 

or

https://www.redcross.ca/donate/appeal/donate-to-the-canadian-red-cross-fund#a0650cb6-ce29-4c40-b663-e749a0a9163f

Note that both these contribution flows have `in Honor of` or `in Memory of` fields. And normally when you click on them a e-card (or sometimes an actual post card) is mailed to the person whom you are honoring. 😄 

I thought this is useful since there were quite a number of situations I've done this; although I don't see a way of doing this at Open Collective. Would love to hear you opinions; if this is something of interest I can open a new issue or add this to our next sprint planning discussion. 😉 

cc: @Betree

**Comment by alanna** [comment]
Sure we could have an "in honor of" field. But for legal/accounting purposes we need to capture the info of the actual donor.
